src/main/drivers/vtx_rtc6705_soft_spi.c

```diff
@@ -20,14 +20,23 @@
 
 #include "platform.h"
 
-#ifdef USE_RTC6705
+#if defined(VTX_RTC6705) && defined(VTX_RTC6705SOFTSPI)
 
 #include "bus_spi.h"
 #include "io.h"
 #include "system.h"
 #include "light_led.h"
 
-#include "vtx_soft_spi_rtc6705.h"
+#include "vtx_rtc6705.h"
+
+#define DP_5G_MASK                  0x7000
+#define PA5G_BS_MASK                0x0E00
+#define PA5G_PW_MASK                0x0180
+#define PD_Q5G_MASK                 0x0040
+#define QI_5G_MASK                  0x0038
+#define PA_BS_MASK                  0x0007
```

Comment:
@fncisco Merging the softspi code and hardware spi code is the topic of a future PR.  Likely much of the code can be unified/deleted/cleaned.  This PR is focused on adding the RTC6705 to the VTX api and porting the features from CF 1.x.